### ISSUE TITLE
Impossible to translate subtitles with video

### ISSUE THREAD

**Issue by rhazor** [SubtitleEdit/subtitleedit#2189]
Video should play the current selected subtitle line once and _Pause_, and not continue playing the rest of the video and start jumping into next empty lines. It's impossible to catch up.

The video should resume/play next subtitle line only after you jump to it with Alt+Arrow Down or Shift+Enter (also Shift+Enter not just jumps to next line, but also triggers Enter in the input and jumps to next row (when inside Text field)).

I have both Auto repeat and Auto continue OFF and 'Select current subtitle while playing' ON. What I understand is that the Auto continue feature does nothing or has no impact, no matter if it's OFF or ON, video just resumes after Shift+Enter or Alt+Arrow up/down and keeps playing and jumping to next untranslated empty lines. Impossible to work with.

You have to keep pressing Ctrl+P to pause each line, on top of that it triggers default Windows system notification sounds for some reason (when inside Text field).

Auto repeat kind of solves it if you set the same line to repeat 9 times until you finish translating current line, but I don't need auto-repeat and also after like 5 minutes of that your head starts to hurt from repeating scenes over and over.

tl:dr video should auto-pause after selected line and resume after manually jumping to next line.

**Comment by niksedk** [comment]
thx for the info :)

It's actually supposed to work like that (pausing in translate mode) and it also used to work like that... could you please check the "translate mode" play/prev/next in this beta build: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.1/SubtitleEditBeta.zip

>also Shift+Enter not just jumps to next line, but also triggers Enter in the input 
Strange, what OS? Does it still do that in the beta?

>`You have to keep pressing Ctrl+P to pause each line, on top of that it triggers default 
>Windows system notification sounds for some reason (when inside Text field).`
Does this beta still do that?

**Comment by rhazor** [comment]
Looks good. Had to adjust shortcuts keys for better workflow.
